doc/api/buffer.markdown

```diff
@@ -577,6 +577,46 @@ for (var key of buf.keys()) {
 //   5
 ```
 
+### buf.lastIndexOf(value[, byteOffset][, encoding])
+
+* `value` {String, Buffer or Number}
```

Comment:
Unfortunately wrapping the three separately confuses the JSON documentation generation IIRC.